.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: Python CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  run_tests:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-20.04
+        python-version:
+          - 3.6
+          - 3.8
+        toxenv: [ py36,py38,quality,docs ]
+        exclude:
+          - python-version: 3.6
+            toxenv: py38
+          - python-version: 3.8
+            toxenv: py36
+          - python-version: 3.8
+            toxenv: quality
+          - python-version: 3.8
+            toxenv: docs
```

Comment:
`quality` and `docs` are passing with `Python 3.8`. We can implement `tox-gh-actions` later on since `Python 3.6` is soon going to be dropped so we won't need to create a matrix for `Python` versions.